src/popover/popover.ts

```diff
@@ -102,12 +84,12 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
    */
   @Input() popoverTitle: string | TemplateRef<any>;
   /**
-   * Placement of a popover accepts:
-   *    "top", "top-left", "top-right", "bottom", "bottom-left", "bottom-right",
-   *    "left", "left-top", "left-bottom", "right", "right-top", "right-bottom"
-   * and array of above values.
-   */
-  @Input() placement: PlacementArray;
+    * Placement of a tooltip accepts:
+    *  - Array or space separated string of following values:
+    *    "top", "top-left", "top-right", "bottom", "bottom-left", "bottom-right",
+    *    "left", "left-top", "left-bottom", "right", "right-top", "right-bottom",
+    */
+  @Input() placement: string | PlacementArray;
```

Comment:
Let's open a separate feature PR for this later for ALL widgets supporting positioning